### ISSUE TITLE
Default to do-not-expose on Mesos/Marathon/Mantl

### ISSUE THREAD

**Issue by josdotso** [containous/traefik#233]
It appears the default being to expose an app via traefik is backward. It should be secure by default, meaning I should opt-into exposing my app to the world via edge, not opt-out.

**Comment by emilevauge** [comment]
Hi @josdotso, we should add a flag `defaultExpose` in marathon config:

```
################################################################
# Mesos/Marathon configuration backend
################################################################

[marathon]
endpoint = "http://localhost:8080/"
networkInterface = "docker0"
watch = true
domain = "marathon.localhost"
defaultExpose = false
```
WDYT?

**Comment by josdotso** [comment]
Sounds good!  Thanks!

**Comment by dylanmei** [comment]
This would be great! It could mean the end of putting `traefik.enable: false` labels in almost everything.

**Comment by emilevauge** [comment]
Fixed by #251
